<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>618</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Умное меню - Добавить блюдо</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="verticalFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>542</width>
      <height>608</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>22</number>
     </property>
     <property name="topMargin">
      <number>11</number>
     </property>
     <property name="rightMargin">
      <number>22</number>
     </property>
     <property name="bottomMargin">
      <number>22</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="addDishHeaderLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">margin-top: 8px;
margin-bottom:8px;
  width: 435px;
  height: 24px;
  color: #2d2c2c;
  font-size: 20px;
  font-family: Inter, &quot;Extra Bold&quot;;
  font-weight: 800;
  text-align: left;
  vertical-align: top</string>
         </property>
         <property name="text">
          <string>Информация о блюде</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="nutritiansHorizontalLayout_2">
         <item>
          <widget class="QLabel" name="unitIngredLoabel_6">
           <property name="minimumSize">
            <size>
             <width>72</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>72</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: none;
color: #2d2c2c;
border:none;
margin-top: 8px;
margin-bottom:8px;

text-align:left;
vertical-align:text-top;
font-size:14px;
font-family:Inter;
line-height:auto;
border-style:hidden;
outline:none;

min-width: 72px;
max-width: 72px;</string>
           </property>
           <property name="text">
            <string>Название:</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameOfDishLineEdit">
           <property name="styleSheet">
            <string notr="true">background-color:#ffffff;
border-radius:4%;
border:2px solid rgba(100, 100, 100, 25);

color: #2d2c2c;

width:360px;
height:32px;

padding-left: 8px;
position:absolute;
text-align:left;
vertical-align:text-top;

font-size:14px;
font-family:Inter;
left:12px;
top:8px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="maxLength">
            <number>32770</number>
           </property>
           <property name="cursorPosition">
            <number>0</number>
           </property>
           <property name="placeholderText">
            <string>Введите название блюда</string>
           </property>
           <property name="cursorMoveStyle">
            <enum>Qt::LogicalMoveStyle</enum>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="infoDishPlainTextEdit">
         <property name="styleSheet">
          <string notr="true">background-color:#ffffff;
border-radius:4%;
border:2px solid rgba(100, 100, 100, 25);

color: #2d2c2c;

width:360px;
height:32px;

padding: 8px;
position:absolute;
text-align:left;
vertical-align:text-top;

font-size:14px;
font-family:Inter;
left:12px;
top:8px;</string>
         </property>
         <property name="plainText">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Введите описание блюда</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="nutritiansHorizontalLayout">
         <item>
          <widget class="QLabel" name="unitIngredLoabel_4">
           <property name="minimumSize">
            <size>
             <width>56</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>56</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: none;
color: #2d2c2c;
border:none;
margin-top: 8px;
margin-bottom:8px;

text-align:left;
vertical-align:text-top;
font-size:14px;
font-family:Inter;
line-height:auto;
border-style:hidden;
outline:none;

min-width: 56px;
max-width: 56px;</string>
           </property>
           <property name="text">
            <string>Ккал:</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="kaloriesLineEdit">
           <property name="styleSheet">
            <string notr="true">background-color:#ffffff;
border-radius:4%;
border:2px solid rgba(100, 100, 100, 25);

color: #2d2c2c;

width:360px;
height:32px;

padding-left: 8px;
position:absolute;
text-align:left;
vertical-align:text-top;

font-size:14px;
font-family:Inter;
left:12px;
top:8px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="cursorPosition">
            <number>0</number>
           </property>
           <property name="placeholderText">
            <string>Введите количество калорий</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="unitIngredLoabel_1">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: none;
color: #2d2c2c;
border:none;
margin-top: 8px;
margin-bottom:8px;

text-align:left;
vertical-align:text-top;
font-size:14px;
font-family:Inter;
line-height:auto;
border-style:hidden;
outline:none;

min-width: 20px;
max-width: 20px;</string>
           </property>
           <property name="text">
            <string>Б:</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="proteinsLineEdit">
           <property name="styleSheet">
            <string notr="true">background-color:#ffffff;
border-radius:4%;
border:2px solid rgba(100, 100, 100, 25);

color: #2d2c2c;

height:32px;

padding-left: 8px;
position:absolute;
text-align:left;
vertical-align:text-top;

font-size:14px;
font-family:Inter;
left:12px;
top:8px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="cursorPosition">
            <number>0</number>
           </property>
           <property name="placeholderText">
            <string>Б:</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="unitIngredLoabel_2">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: none;
color: #2d2c2c;
border:none;
margin-top: 8px;
margin-bottom:8px;

text-align:left;
vertical-align:text-top;
font-size:14px;
font-family:Inter;
line-height:auto;
border-style:hidden;
outline:none;

min-width: 20px;
max-width: 20px;</string>
           </property>
           <property name="text">
            <string>Ж:</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fatsLineEdit">
           <property name="styleSheet">
            <string notr="true">background-color:#ffffff;
border-radius:4%;
border:2px solid rgba(100, 100, 100, 25);

color: #2d2c2c;

height:32px;

padding-left: 8px;
position:absolute;
text-align:left;
vertical-align:text-top;

font-size:14px;
font-family:Inter;
left:12px;
top:8px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="cursorPosition">
            <number>0</number>
           </property>
           <property name="placeholderText">
            <string>Ж:</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="unitIngredLoabel_3">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: none;
color: #2d2c2c;
border:none;
margin-top: 8px;
margin-bottom:8px;

text-align:left;
vertical-align:text-top;
font-size:14px;
font-family:Inter;
line-height:auto;
border-style:hidden;
outline:none;

min-width: 20px;
max-width: 20px;</string>
           </property>
           <property name="text">
            <string>У:</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="carbohLineEdit">
           <property name="styleSheet">
            <string notr="true">background-color:#ffffff;
border-radius:4%;
border:2px solid rgba(100, 100, 100, 25);

color: #2d2c2c;

height:32px;

padding-left: 8px;
position:absolute;
text-align:left;
vertical-align:text-top;

font-size:14px;
font-family:Inter;
left:12px;
top:8px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="cursorPosition">
            <number>0</number>
           </property>
           <property name="placeholderText">
            <string>У:</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="ingredientsHeaderLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">margin-top: 8px;
margin-bottom:8px;
  width: 435px;
  height: 24px;
  color: #2d2c2c;
  font-size: 20px;
  font-family: Inter, &quot;Extra Bold&quot;;
  font-weight: 800;
  text-align: left;
  vertical-align: top</string>
         </property>
         <property name="text">
          <string>Ингредиенты:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="prodListTableView">
         <property name="styleSheet">
          <string notr="true">QTableView{
margin-top: 8px;
background-color: rgba(255, 255, 255, 50);
border: 3px solid rgba(255, 255, 255, 75);
border-top-left-radius: 0%;
border-top-right-radius: 0%;
}

QTableView::section {
background-color: rgba(55, 55, 55);
color: white;
border: none;
}

QTableView::item {
border-style: none;
border-bottom: rgba(255, 255, 255, 50);
}

QTableView::item:selected {
border-style: none;
border-bottom: rgba(255, 255, 255, 50);
}</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="actionsStorageHorizontalLayout">
         <item>
          <widget class="QPushButton" name="addDishPushButton">
           <property name="styleSheet">
            <string notr="true">QPushButton{
height:32px;
width:auto;
margin-top: 16px;

color: #30353B;
background: #ADDC81;

text-align:center;
vertical-align:center;
font-size:12px;
font-family:Inter;
line-height:auto;
font-weight: bold;

border-radius:4%;
border: 2 px solid #ADFF81;
}

QPushButton:hover {
background: #AAAA81;
}

QPushButton:pressed{
background: #AAAAFF;
}


</string>
           </property>
           <property name="text">
            <string>Сохранить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelDishPushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
height:32px;
width:auto;

color: #30353B;
background: #ADDC81;
margin-top: 16px;

text-align:center;
vertical-align:center;
font-size:12px;
font-family:Inter;
line-height:auto;
font-weight: bold;

border-radius:4%;
}

QPushButton:hover {
background: #AAAA81;
border: 2px solid rgba(146, 146, 120, 75);
}

QPushButton:pressed{
background: #AAAAFF;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../icons/bx_trash.svg</normaloff>../icons/bx_trash.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
